runtime(rasi): add filetype and syntax plugin

Fymyte opened the pull request:
ported from: https://github.com/Fymyte/rasi.vim

Signed-of-by: Pierrick Guillaume <pierguill@gmail.com>

chrisbra commented on runtime/ftplugin/rasi.vim:
```diff
@@ -0,0 +1,25 @@
+
+let b:undo_ftplugin = "setl com< cms< isk<"
+
+setlocal comments=s1:/*,mb:*,ex:*/ commentstring&
```
no need to reset `commentstring` option, since you are setting it in the next line anyhow.

chrisbra commented on runtime/ftplugin/rasi.vim:
```diff
@@ -0,0 +1,25 @@
+setlocal commentstring=//\ %s
+setlocal iskeyword+=-
+
+let &l:include = '^\s*@import\s\+\%(url(\)\='
```
missing in `b:undo_ftplugin`

Fymyte committed:
runtime(rasi): add filetype and syntax plugin

ported from: https://github.com/Fymyte/rasi.vim

Signed-of-by: Pierrick Guillaume <pierguill@gmail.com>
